recipes/quatradis/meta.yaml

```diff
@@ -1,29 +1,38 @@
-{% set version="0.8.3" %}
+{% set version="1.0.1" %}
 
 package:
   name: quatradis
   version: {{ version }}
 
 source:
   url: https://github.com/quadram-institute-bioscience/QuaTradis/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 020b8f25aab24a18ca1d9c810fa3d455c1b2328d4f634126a41dabede6cf44ec
+  sha256: 8ff35ebb615d6ab4698f3204d0517c432ac987cb4f570b1b0a0d2d2fadb73af4
 
 build:
   number: 0
-  noarch: python
+  skip: True
```

Comment:
this will always skip the recipe (i.e., never build it!)...tests are passing because the build system thinks there's nothing to do and exists cleanly.